deploy/docker/prestart.sh

```diff
@@ -1,22 +1,31 @@
 #!/bin/bash
 
 RANDOM_ADMIN_PASS=`python -c "import secrets;chars = 'abcdefghijklmnopqrstuvwxyz0123456789';print(''.join(secrets.choice(chars) for i in range(10)))"`
-ADMIN_PASSWORD=${ADMIN_PASSWORD:-$RANDOM_ADMIN_PASS}
+ADMIN_PASSWORD=${RANDOM_ADMIN_PASS:-$ADMIN_PASSWORD}
```

Comment:
This would render ADMIN_PASSWORD redundant, remove it also remove it from Dockerfile